docs/recommendation.en-US.md

```diff
@@ -48,4 +48,4 @@ State Management|[ngrx](https://ngrx.io/) | NGRX state management
description">A powerful Axure library</p></a></div><div class="ant-col ant-col-xs-24 ant-col-sm-12 ant-col-md-8 ant-col-lg-8 gutter-row" style="padding:12px"><a class="resource-card" target="_blank" rel="noopener" href="http://kitchen.alipay.com"><img class="resource-card-image" src="https://gw.alipayobjects.com/zos/basement_prod/d475d063-2754-4442-b9db-5d164e06acc9.svg" alt="Kitchen"><p class="resource-card-title">Kitchen</p><p class="resource-card-description">A Sketch plugin collection</p></a></div><div class="ant-col ant-col-xs-24 ant-col-sm-12 ant-col-md-8 ant-col-lg-8 gutter-row" style="padding:12px"><a class="resource-card" target="_blank" rel="noopener" href="https://landing.ant.design/docs/download-cn"><img class="resource-card-image" src="https://gw.alipayobjects.com/zos/basement_prod/b443f4be-5116-49b7-873f-a7c8502b8f0e.svg" alt="Ant Design Landing"><p class="resource-card-title">Ant Design Landing</p><p class="resource-card-description">Landing Templates</p></a></div><div class="ant-col ant-col-xs-24 ant-col-sm-12 ant-col-md-8 ant-col-lg-8 gutter-row" style="padding:12px"><a class="resource-card" target="_blank" rel="noopener" href="http://ux.ant.design"><img class="resource-card-image" src="https://gw.alipayobjects.com/zos/basement_prod/51682163-e01a-46fe-810c-ac0062379717.svg" alt="Ant UX"><p class="resource-card-title">Ant UX</p><p class="resource-card-description">A set of page logic prototype libraries</p></a></div><div class="ant-col ant-col-xs-24 ant-col-sm-12 ant-col-md-8 ant-col-lg-8 gutter-row" style="padding:12px"><a class="resource-card" target="_blank" rel="noopener" href="https://www.xiaopiu.com/topic/ant-design"><img class="resource-card-image" src="https://gw.alipayobjects.com/zos/basement_prod/77e6a9ae-24a9-4be6-be42-f7fa8ee0eecf.svg" alt="Ant Design Prototype (xiaopiu)"><p class="resource-card-title">Ant Design Prototype (xiaopiu)</p><p class="resource-card-description">Online library and prototype</p></a></div><div class="ant-col ant-col-xs-24 ant-col-sm-12 ant-col-md-8 ant-col-lg-8 gutter-row" style="padding:12px"><a class="resource-card" target="_blank" rel="noopener" href="https://www.antforfigma.com"><img class="resource-card-image" src="https://gw.alipayobjects.com/zos/basement_prod/7b9ed3f2-6f05-4ddb-bac3-d55feb71e0ac.svg" alt="UI Kit for Figma"><p class="resource-card-title">UI Kit for Figma</p><p class="resource-card-description">Library of components for Desktop</p></a></div><div class="ant-col ant-col-xs-24 ant-col-sm-12 ant-col-md-8 ant-col-lg-8 gutter-row" style="padding:12px"><a class="resource-card" target="_blank" rel="noopener" href="https://gw.alipayobjects.com/os/basement_prod/862ee74f-4ac5-482c-b1ae-3165684cedbe.sketch"><img class="resource-card-image" src="https://gw.alipayobjects.com/zos/basement_prod/a9dc586a-fe0a-4c7d-ab4f-f5ed779b963d.svg" alt="Ant Design Chart"><p class="resource-card-title">Ant Design Chart</p><p class="resource-card-description">Sketch Symbols for Chart</p></a></div></div>
```

Comment:
Hi @trungk18 
it seems this line was unexpected formatted, could you fix this?

Reply:
I did the web edit. Not sure why it resulted on that change. Let me fix it and squash them together.